components/sysutils/nut/nut-bins.p5m

```diff
@@ -0,0 +1,111 @@
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)/bins@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - NUT Command-line binaries and scripts"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+
+depend fmri=$(COMPONENT_FMRI)-common@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION) type=require
+
+### NUT Command-line binaries and scripts (daemons, clients)
+
+### Note: While nut-scanner can use snmp, neon/xml, ipmi and usb libs, it pulls
+### them truly dynamically (if available) and skips otherwise. Installing all
+### the NUT driver packages should pull those libs by harder ldd requirements.
+### Conversely if you don't want those UPSes (got no drivers) why scan them? :)
+
+file path=usr/bin/$(MACH64)/nut-scanner
+file path=usr/bin/$(MACH64)/upsc
+file path=usr/bin/$(MACH64)/upscmd
+file path=usr/bin/$(MACH64)/upslog
+file path=usr/bin/$(MACH64)/upsrw
+file path=usr/bin/$(MACH64)/upssched-cmd
+
+file path=usr/bin/nut-scanner
+file path=usr/bin/upsc
+file path=usr/bin/upscmd
+file path=usr/bin/upslog
+file path=usr/bin/upsrw
+file path=usr/bin/upssched-cmd
+
+file path=usr/sbin/$(MACH64)/upsd
+file path=usr/sbin/$(MACH64)/upsdrvctl
+file path=usr/sbin/$(MACH64)/upsmon
+file path=usr/sbin/$(MACH64)/upssched
+
+file path=usr/sbin/upsd
+file path=usr/sbin/upsdrvctl
+file path=usr/sbin/upsmon
+file path=usr/sbin/upssched
+
+# Example notification script for UPSMON
+file files/notifyme path=$(NUTDRVDIR)/notifyme
+
+file path=usr/share/man/man5/hosts.conf.5
+file path=usr/share/man/man5/nut.conf.5
+file path=usr/share/man/man5/ups.conf.5
+file path=usr/share/man/man5/upsd.conf.5
+file path=usr/share/man/man5/upsd.users.5
+file path=usr/share/man/man5/upsmon.conf.5
+file path=usr/share/man/man5/upssched.conf.5
+
+file path=usr/share/man/man8/nut-scanner.8
+file path=usr/share/man/man8/upsc.8
+file path=usr/share/man/man8/upscmd.8
+file path=usr/share/man/man8/upsd.8
+file path=usr/share/man/man8/upsdrvctl.8
+file path=usr/share/man/man8/upslog.8
+file path=usr/share/man/man8/upsmon.8
+file path=usr/share/man/man8/upsrw.8
+file path=usr/share/man/man8/upssched.8
+
+# NUT starter examples are by convention across many distros delivered in /etc
+file path=$(NUTETCDIR)/hosts.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0640
```

Comment:
Is it a bad thing? Users get current-version samples in the same dir, so
can later easily inspect (diff) if their configs use all modern keywords
etc.

--
Typos courtesy of GMail on my Samsung Android
15.05.2016 7:04 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/sysutils/nut/nut-bins.p5m
> <https://github.com/OpenIndiana/oi-userland/pull/2001#discussion_r63286253>
> :
>
> > +file path=usr/share/man/man5/upsd.users.5
> > +file path=usr/share/man/man5/upsmon.conf.5
> > +file path=usr/share/man/man5/upssched.conf.5
> > +
> > +file path=usr/share/man/man8/nut-scanner.8
> > +file path=usr/share/man/man8/upsc.8
> > +file path=usr/share/man/man8/upscmd.8
> > +file path=usr/share/man/man8/upsd.8
> > +file path=usr/share/man/man8/upsdrvctl.8
> > +file path=usr/share/man/man8/upslog.8
> > +file path=usr/share/man/man8/upsmon.8
> > +file path=usr/share/man/man8/upsrw.8
> > +file path=usr/share/man/man8/upssched.8
> > +
> > +# NUT starter examples are by convention across many distros delivered in /etc
> > +file path=$(NUTETCDIR)/hosts.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0640
>
> One issue with pkg vs , for example, apt-get: when you have example files,
> with apt-get you can do mv /etc/file.sample /etc/file, but IPS will fix it
> back, so you'll get all "sample" files back on pkg fix. So I'd prefer to
> get them separate from actual config files.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/2001/files/8763ecacc8d248aa64f9eaebf64c307267ff7ab5#r63286253>
>